src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -50,4 +52,23 @@ public String sendNotificationEmail(String emailAddress, String forename, String
             return "Email request failed to send: " + emailAddress;
         }
     }
+
+    /**
+     * Method which sends a request to publication services to email the P&I team.
+     * With a list of all media applications in the database.
+     *
+     * @return String for logging success or failure
+     */
+    public String sendMediaApplicationReportingEmail(List<MediaApplication> mediaApplicationList) {
+        try {
+            return webClient.post().uri(url + "/notify/media/report")
+                .body(BodyInserters.fromValue(mediaApplicationList)).retrieve()
+                .bodyToMono(String.class).block();
+        } catch (WebClientException ex) {
+            log.error(String.format("Request: %s failed. With error message: %s",
+                                    mediaApplicationList, ex.getMessage()));
+            return String.format("Email request failed to send with list of applications: %s",
+                                 mediaApplicationList);
```

Comment:
this is the same thing, youre logging the result in the caller service too, you only need to do 1 of these

Reply:
Done